### ISSUE TITLE
Chat detection broke in 1.29.2

### ISSUE THREAD

**Issue by ghost** [AucT/AucT-Hotkeys-Tool#4]
Looks like patch 1.29.2 (https://us.battle.net/forums/en/bnet/topic/20763816274) broke chat detection. I'm using AucT v2.9.6.

**Comment by AucT** [comment]
released v2.9.7. Respond if it's working normally for you.

**Comment by ghost** [comment]
Still broken for me. Here is my .ini config file:
```
[Inventory]
item2=T
item1=R
item3=D
item4=F
item5=Space
item6=WheelUp
[Others]
Scoreboard=
Hero=1
Share=
Ignore=
AHP=
Garena=
toggle=F8
EHP=
DontShowConfig=0
DontShowConfigTemp=0
WMC=0
profile=Crusader
pause=
chat=1
[CustomKeys]
EnSkills=0
SelfCast=0
SmartSkills=0
skill6=
skill7=
skill8=Space
skill2=
skill1=
skill3=
skill4=
NewWarCraft=1
[Messages]
hotkey1=            
message1=            
hotkey2=         
message2=         
hotkey3=       
message3=       
hotkey4=^M      
message4=-zoom 900      
hotkey= 
message= 
hotkey5=  
message5=  
hotkey6=    
message6=    
hotkey7=    
message7=    
hotkey8=  
message8=  
hotkey9= 
message9= 
hotkey10=  
message10=  
hotkey11=  
message11=  
hotkey12= 
message12= 
[RemapKey]
KRhotkey1=Q  
KRvalue1=C  
KRhotkey2=W  
KRvalue2=D  
KRhotkey3=E   
KRvalue3=F   
KRhotkey= 
KRvalue= 
KRhotkey4=C  
KRvalue4=  
KRhotkey5= 
KRvalue5= 
KRhotkey6= 
KRvalue6= 
```

**Comment by AucT** [comment]
Weird stuff happens with new patch.
I can say only that on my win 10, wc3 1.29.2.9231, and aht 2.9.7 it is working (in custom game an local game).

You can do:
Try to run as admin AHT
Ensure aht has picked up your config, or copy fresh AHT in new folder and check New WarCraft UI and set some keys for test.

If nothing works there is Other hotkey tool https://sih.auct.eu/, it is uses other approach that press backspace after hotkey. If you don't play unofficial dota it should work even better.

There are also other hotkeys like garenatotal and vaphotkey and others.

**Comment by AucT** [comment]
found typo. check 2.9.8

**Comment by ghost** [comment]
Confirmed it's fixed in 2.9.8. Thank you!
